lib/plugins/aws/package/compile/events/websockets/lib/validate.test.js

```diff
@@ -268,3 +269,42 @@ describe('#validate()', () => {
       .with.length(0);
   });
 });
+
+describe('#validate() using runServerless util', () => {
+  it('should use provided authorizer name when name field is supplied', () => {
+    const nameField = 'authName';
+    const arnName = 'auth';
+
+    runServerless({
```

Comment:
Let's use async/await (note also that here promise as being not returned is orphaned)

Reply:
@medikoo just changed to use async/await. This explains some issues I was having earlier. Thanks for pointing out! 

Also, I noticed all of the test examples given in the [test docs](https://github.com/serverless/serverless/tree/master/test#unit-tests) use `.then()' instead of async/await and they don't return a promise either (e.g. [here](https://github.com/serverless/serverless/blob/74634c3317a116077a008375e20d6a5b99b1256e/lib/plugins/aws/package/compile/functions/index.test.js#L2605-L2608)). 

Are we trying to stick to async/await moving forward? If so, I can open a PR that adds a note to the test docs to that effect.

Reply:
@zaldanaraul as I look they return the promise (note no brackets wrapping arrow function body).

There's no async/await as they were written before we could use it before breaking things. While it be nice to have everything migrated to async/await, it's not easily doable, do we do it on demand

Reply:
you're right. Just noticed the missing brackets. My bad!